### PR TITLE
add Alluxio example

### PR DESCRIPTION
**Pull request by saveveltri**

### CODE REVIEW

File: snippets/include-alluxio.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<dependency>
```
Comment (stefanobaghino) [inline snippets/include-alluxio.xml]: Would it be feasible to refer directly to the `pom.xml`?

File: snippets/rbp-repository.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<repositories>
```
Comment (stefanobaghino) [inline snippets/rbp-repository.xml:1]: Would it be feasible to refer directly to the `pom.xml`?

**Comment by saveveltri** [comment]
Thanks Stefano for the review. One point. Addressing your comments would remove redundant files, but, on the other hand, it means that every updates on the source file (the pom.xml in this case) can affect all the doc files it is referred. It could get tough to handle this.
What is your thought?

Thanks

**Comment by stefanobaghino** [comment]
@saveveltri The safest way to use the `pom.xml` without affecting other pages is to append your edits at the end, as long as you don't push other lines down there's no risk. I did the same for [this commit](https://github.com/radicalbit/rbp-docs-code/pull/5/files#diff-02bea068fb725286d736b044a68d9d17L114). We can check how it turns out this way tomorrow, if you think the change is feasible. 😉

**Comment by saveveltri** [comment]
An ad hoc project for Alluxio API has been created based on our morning thoughts

**Comment by stefanobaghino** [comment]
Very good, is it safe to remove code under `mvn` now? Or is it still referenced in the documentation?

**Comment by saveveltri** [comment]
if no one else is currently using code under `mvn` I will gladly remove it, otherwise we could arrange a `clean up` PR @alkagin @maocorte @giampaolotrapasso

**Comment by alkagin** [comment]
@saveveltri I've already open a PR to move the connectors examples from mvn folder to independent projects, but I still prefer a clean up PR to avoid a huge rebase in the opened PR.

**Comment by stefanobaghino** [comment]
@saveveltri I think you can safely remove the duplicate examples you introduced with this PR under `mvn` without affecting others' work.

**Comment by giampaolotrapasso** [comment]
Thanks

**Pull request merged**